SimG4Core/PrintGeomInfo/test/python/g4OverlapCheckDD4Hep_cfg.py

```diff
@@ -0,0 +1,37 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.Eras.Era_Run3_dd4hep_cff import Run3_dd4hep
+
+process = cms.Process("G4PrintGeometry",Run3_dd4hep)
+
+process.load('Configuration.Geometry.GeometryDD4hepExtended2021_cff')
+
+from SimG4Core.PrintGeomInfo.g4TestGeometry_cfi import *
+process = checkOverlap(process)
+
+# enable Geant4 overlap check 
+process.g4SimHits.CheckGeometry = True
+
+# Geant4 geometry check 
+process.g4SimHits.G4CheckOverlap.OutputBaseName = cms.string("cms2021dd4hep")
+process.g4SimHits.G4CheckOverlap.OverlapFlag = cms.bool(True)
+process.g4SimHits.G4CheckOverlap.Tolerance  = cms.double(0.01)
+process.g4SimHits.G4CheckOverlap.Resolution = cms.int32(10000)
+process.g4SimHits.G4CheckOverlap.Depth      = cms.int32(-1)
+# tells if NodeName is G4Region or G4PhysicalVolume
+process.g4SimHits.G4CheckOverlap.RegionFlag = cms.bool(False)
+# list of names
+process.g4SimHits.G4CheckOverlap.NodeNames  = cms.vstring('OCMS')
+process.g4SimHits.G4CheckOverlap.Tolerance  = cms.double(1)
```

Comment:
@bsunanda , you may want remove this line.

Reply:
Done that